pandas/core/groupby.py

```diff
@@ -1476,8 +1476,93 @@ def describe(self, **kwargs):
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
         """
-        Provide resampling when using a TimeGrouper
-        Return a new grouper with our resampler appended
+        Provide resampling when using a TimeGrouper.
+
+        Given a grouper the function resamples it according to a string and an
+        optional list and dictionary of parameters. Returns a new grouper with
+        our resampler appended.
+
+        Parameters
+        ----------
+        rule : str
+            The offset string or object representing target grouper conversion.
+        *args
```

Comment:
This feels a bit too specific to the implementation to me. Do you have any specific args / kwargs that the user would care about? Otherwise, I'd just write it as

```
*args, **kwargs
    For compatibility with other groupby methods.
```

Reply:
Some keywords to think about (explicitly document and show examples for)?

```
closed : {'right', 'left'}
label : {'right', 'left'}
loffset : timedelta
```

See pandas.core.generic.NDFrame.resample for some others that may or may not have an effect. I'm not sure.